src/Core/Utilities/Plan/CustomPlan.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Bit.Core.Enums;
+
+namespace Bit.Core.Utilities.Plan;
+
+public record CustomPlan : Models.StaticStore.Plan
+{
+    public CustomPlan()
+    {
+        Type = PlanType.Custom;
+        PasswordManager = new CustomPasswordManagerFeatures();
+    }
+
+    private record CustomPasswordManagerFeatures : PasswordManagerPlanFeatures
+    {
+        public CustomPasswordManagerFeatures()
+        {
+            AllowSeatAutoscale = false;
```

Comment:
Does not match the current value in `PasswordManagerPlanStore`

```suggestion
            AllowSeatAutoscale = true;
```